app/javascript/components/search/genes/GeneSearchView.js

```diff
@@ -27,7 +27,7 @@ export default function GeneSearchView() {
                                  !geneSearchState.isLoading &&
                                  !geneSearchState.isError
 
-  let geneSearchPlaceholder = 'Search for genes across all studies'
+  let geneSearchPlaceholder = 'genes (e.g. "agpat pten")'
```

Comment:
[Searching genes for AGPAT](https://singlecell.broadinstitute.org/single_cell/app/genes?type=study&page=1&genes=AGPAT&genePage=1) returns no results -- a functional issue with using this as an example.  

Gene NF2, which has results and interacts with PTEN, would fit better:
```suggestion
  let geneSearchPlaceholder = 'genes (e.g. "PTEN NF2")'
```